examples/workflow/automatic_speech_recognition/automatic_speech_recognition/utils.py

```diff
@@ -89,13 +89,13 @@ def generate_diff_word_level(reference: str, candidate: str) -> Dict[str, Any]:
     }
 
 
-def preprocess_transcription(txt: Union[GroundTruth, Inference]) -> str:
+def preprocess_transcription(txt: str) -> str:
     """
     Preprocesses and standardizes text to prepare for metrics evaluations.
     Removes punctuation, changes text to lower case, and converts all NumWords to Numbers.
     """
     num = NumWordsToNum()
-    txt = re.sub(r"[^\w\s]", "", txt.transcription.label.lower())
+    txt = re.sub(r"[^\w\s]", "", txt)
```

Comment:
based on the usage of `preprocess_transcription`(examples/workflow/automatic_speech_recognition/automatic_speech_recognition/evaluator.py), i don't think we should modify the code at here

```py
def compute_test_sample_metrics(gt: GroundTruth, inf: Inference) -> TestSampleMetric:
    gt = preprocess_transcription(gt)
    inf = preprocess_transcription(inf)
```